<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectDetectionView</class>
 <widget class="QWidget" name="ObjectDetectionView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ObjectDetectionView</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QGraphicsView" name="graphicsView">
     <property name="minimumSize">
      <size>
       <width>640</width>
       <height>480</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_weight">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_Cfg"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Weights</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Names</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Cfg:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_cfg">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_Weight"/>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="lineEdit_Name"/>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="pushButton_names">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Camera</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="comboBox_Camera"/>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="pushButton_switch">
        <property name="text">
         <string>switch</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cfg</sender>
   <signal>clicked()</signal>
   <receiver>ObjectDetectionView</receiver>
   <slot>OnBrowseCfg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>1061</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_weight</sender>
   <signal>clicked()</signal>
   <receiver>ObjectDetectionView</receiver>
   <slot>OnBrowseWeights()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1005</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_names</sender>
   <signal>clicked()</signal>
   <receiver>ObjectDetectionView</receiver>
   <slot>OnBrowseNames()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>994</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>705</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_switch</sender>
   <signal>clicked()</signal>
   <receiver>ObjectDetectionView</receiver>
   <slot>OnSwitch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1010</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnBrowseCfg()</slot>
  <slot>OnBrowseWeights()</slot>
  <slot>OnBrowseNames()</slot>
  <slot>OnSwitch()</slot>
 </slots>
</ui>
